packages/compass-aggregations/src/utils/get-directory.js

```diff
@@ -4,6 +4,7 @@ let electronApp;
 try {
   electronApp = require('@electron/remote').app;
 } catch (e) {
+  // eslint-disable-next-line no-console
```

Comment:
For most files where there are only one or two console lines I just disabled the line. For files with a bunch (often tests) I disabled it for the whole file. For packages that exist basically to console.log (scripts, build packages) I just disabled the rule in its eslint rc file.